app/src/main/res/values-fr/strings.xml

```diff
@@ -45,7 +45,7 @@
   <string name="family_name">Nom de famille</string>
   <string name="add_user_example_family_name">Smith</string>
   <string name="action_search">Rechercher</string>
-  <string name="action_go_to">ID</string>
+  <string name="action_go_to">Recherche ID</string>
```

Comment:
Not sure, but maybe 'Rechercher par ID' would be better. It worths double check it.